<launch>
    <node pkg="rb5_vision" exec="yolo_camera_node_script.py" name="yolo_camera_node">
        <param name="camera_id" value="0"/>
        <param name="topic_name" value="/camera_0"/>
        <param name="frame_rate" value="30"/>
        <param name="object_file" value="/../../rb5_control/object.txt"/>
    </node>

    <node pkg="rb5_control" exec="waypoint_navigator_script.py" name="waypoint_navigator">
        <param name="waypoint_file" value="/path/to/your/waypoints.txt"/>
    </node>
</launch>
